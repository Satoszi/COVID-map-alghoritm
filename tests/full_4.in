20 20 2 76
full_4
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
++++++++++++++++++++
