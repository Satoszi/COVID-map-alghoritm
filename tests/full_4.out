19 19
15 19
12 19
9 19
6 19
3 19
0 19
17 18
13 17
8 17
4 17
1 17
19 16
16 16
11 16
6 16
14 15
9 15
3 15
0 15
18 14
12 14
7 14
16 13
10 13
5 13
2 13
19 12
14 12
8 12
0 12
17 11
12 11
6 11
3 11
15 10
10 10
1 10
19 9
13 9
8 9
5 9
17 8
11 8
3 8
0 8
15 7
9 7
6 7
19 6
13 6
4 6
1 6
17 5
11 5
8 5
15 4
6 4
3 4
0 4
19 3
12 3
9 3
17 2
14 2
7 2
4 2
1 2
11 1
19 0
16 0
13 0
9 0
6 0
3 0
0 0
